spec/services/github/contributions/create_spec.rb

```diff
@@ -46,6 +48,11 @@
         call_create
       end
 
+      it 'associates users with contributions' do
+        expect(ContributionsUser).to receive(:create).twice
+        call_create
```

Comment:
call the subject directly here, it will be more explicit

Reply:
In this case `call_create` is the subject, it would be better to replace by `subject` only?